themes/SuiteP/include/EditView/actions_buttons.tpl

```diff
@@ -7,7 +7,7 @@
         {{sugar_button module="$module" id="SAVE" view="$view" form_id="$form_id"}}
         {{sugar_button module="$module" id="CANCEL" view="$view" form_id="$form_id" }}
     {{/if}}
-    {if $showVCRControl}
```

Comment:
We shouldn't be putting in template fixes that is very specific (don't want to use the term 'hack').  Need to have this resolved at a higher level i.e. before the template is parsed.